insights/parsers/sys_fs_cgroup_memory_tasks_number.py

```diff
@@ -0,0 +1,38 @@
+"""
+SysFsCgroupMemoryTasksNumber - Command ``/usr/bin/find /sys/fs/cgroup/memory -name 'tasks'``
+============================================================================================
+
+This parser reads the file number from the output of ``/usr/bin/find /sys/fs/cgroup/memory -name 'tasks'``.
+
+"""
+
+from insights import Parser, parser
+from insights.parsers import SkipException
+from insights.specs import Specs
+
+
+@parser(Specs.sys_fs_cgroup_memory_tasks_number)
+class SysFsCgroupMemoryTasksNumber(Parser, list):
+    """
+    Class ``SysFsCgroupMemoryTasksNumber`` parses the file number from the output of the
```

Comment:
@wushiqinlou - Would you please update the docstring more relevant? Since it parser the datasource (count of tasks) but not the command output.